code/web/src/modules/user/Profile.js

```diff
@@ -9,54 +9,233 @@ import { Link } from 'react-router-dom'
+    createdAt:"1616449062163",
+    updatedAt:"1616449062163",
+    kept: false
+  }
+]
 
 // Component
-const Profile = (props) => (
-  <div>
-    {/* SEO */}
-    <Helmet>
-      <title>My Profile - Crate</title>
-    </Helmet>
-
-    {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H3 font="secondary">My profile</H3>
-      </GridCell>
-    </Grid>
-
-    <Grid>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-
-        <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
-
-        <Link to={userRoutes.subscriptions.path}>
-          <Button theme="primary">Subscriptions</Button>
-        </Link>
-
-        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
-      </GridCell>
-    </Grid>
-  </div>
-)
+class Profile extends PureComponent {
+
+  constructor() {
+    super()
+  }
+
+  // Runs on server only for SSR
+  static fetchData({ store }) {
+    return store.dispatch(getListByUser())
+  }
+
+  // Runs on client only
+  componentDidMount() {
+    this.props.getListByUser()
+  }
+
+  render() {
+    const products = mockProductHistory.map((product, i) => {
+      return (
+        <GridCell key={i}>
+          <Card style={{ width: '18em', backgroundColor: white, margin: 'auto' }}>
+            <p style={{ padding: '2em 3em 0 3em' }}>
+              <img src={routeImage + product.image} alt={product.slug} style={{ width: '100%' }}/>
+            </p>
+
+            <div style={{ padding: '1em 1.2em' }}>
+              <H4 font="secondary" style={{ color: black }}>{ product.name }</H4>
+              <p>Description: {product.description}</p>
+              <p>Order Number: {product.createdAt}</p>
+              <p>Product Status: {product.kept ? "Kept" : "Returned"}</p>
+            </div>
+          </Card>
+        </GridCell>
+      )
+    })
+
+    return (
+      <div>
+        {/* SEO */}
+        <Helmet>
+          <title>My Profile - Crate</title>
+        </Helmet>
+
+        {/* Top title bar */}
+        <Grid style={{ backgroundColor: grey }}>
+          <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+            <H3 font="secondary">My profile</H3>
+          </GridCell>
+        </Grid>
+
+        {/* User information */}
+        <Grid justifyCenter={true} alignCenter={true}>
+          <GridCell style={{
+              marginLeft: '2em',
+              padding: '2em',
+              textAlign: 'center',
+              objectFit: 'contain',
+              maxWidth: 250,
+              maxHeight: 250
+            }}>
+            <ImageTile height={250} width={250} image={userImage} shadow={level3}/>
+          </GridCell>
+
+          <GridCell>
+            <H4 style={{ marginBottom: '0.5em' }}>{this.props.user.details.name}</H4>
```

Comment:
Once we add the conditional rendering, there will be more here - to show the form instead of just displaying the user's data.